content/en/client/token.md

```diff
@@ -48,21 +56,28 @@ curl -X POST \
 
 Note the following:
 
-* `client_id` and `client_secret` were provided in the response text when you registered your application.
-* `redirect_uri` must be one of the URIs defined when registering the application.
-* We are requesting a `grant_type` of `client_credentials`, which defaults to giving us the `read` scope.
+- `client_id` and `client_secret` were provided in the response text when you registered your application.
+- `redirect_uri` must be one of the URIs defined when registering the application.
+- We are requesting a `grant_type` of `client_credentials`, which defaults to giving us the `read` scope.
+
+The response of this method is a [Token]({{< relref "entities/token" >}}) entity. We will need the `access_token` value. Once you have the access token, save it in your local cache.
 
-The response of this method is a [Token]({{< relref "entities/token" >}}) entity. We will need the `access_token` value. Once you have the access token, save it in your local cache. To use it in requests, add the HTTP header `Authorization: Bearer ...` to any API call that requires OAuth (i.e., one that is not publicly accessible). Let's verify that our obtained credentials are working by calling [GET /api/v1/apps/verify_credentials]({{< relref "methods/apps#verify_credentials" >}}):
+{{< hint style="warning" >}}
+Treat the `access_token` as if it were a password. We recommend you encrypt this value when storing in your cache, to prevent accidental credential exposure.
+{{< /hint >}}
+
+To use it in requests, add the HTTP header `Authorization: Bearer <access_token>` to any API call that requires OAuth (i.e., one that is not publicly accessible).
```

Comment:
Maybe slightly out of scope of this PR, but may be useful to a do a once-over for consistent usage of this placeholder value. You are using `<access_token>` here (which makes complete sense to me). In an example below there's a place that uses "our_access_token_here" for example. Maybe there are more differences elsewhere. Would be nice for consistency to align them.

Reply:
Okay, have done this.